src/strategy/mod.rs

```diff
@@ -92,7 +92,13 @@ impl UpdateStrategy {
                 "{}, total schedule length {} minutes (next window {})",
                 self.configuration_label(),
                 p.schedule_length_minutes(),
-                p.human_remaining()
+                // If not using UTC, we cannot accurately calculate remaining time;
+                // display next window week day and time, instead.
+                if p.use_system_time {
+                    p.human_next_window()
```

Comment:
`human_next_window()` attempts to return the week day and time of the next nearest reboot window. However, the current implementation _may_ lead to an incorrect next nearest reboot window if the function is called in the week of a "DST shift" (e.g. in Eastern Time, the week of "spring forward" or "fall back" day).

Reply:
IMHO This is seem good enough already as you did. Do you think we should maybe consider doing `human_next_window()` unconditionally instead?

Reply:
I think `human_next_window()` is easier to understand if the reader is in the same time zone as the configured time zone (likely the case when using a non-UTC time zone). But if the user is in a different time zone than the configured time zone (likely the case when using UTC), `human_next_window()` might require some careful mental gymnastics, especially if the reboot window is near.
TL;DR, I think `human_remaining()` is still useful if we're using UTC.